Let LaTeX with XeTeX/LuaTeX use normal sized typeface in code-blocks

jfbu opened the pull request:
closes: #4791

I have tested normal size typeface in codeblocks with xelatex as latex engine for a build of the v1.0.0 SciPy docs, and it looks much better with normal size than small size, because the monospaced font is already of the same design family as the font for main text.

jfbu committed:
Let LaTeX with XeTeX/LuaTeX not use small typeface in code-blocks

closes: #4791

tk0miya commented on doc/config.rst:
```diff
@@ -1998,7 +1998,16 @@ These options influence LaTeX output. See further :doc:`latex`.
+        ``'\\fvset{fontsize=auto}'``, because the default fonts are part of
+        one unified typeface family (Latin Modern OpenType).
+
+        .. versionadded:: 1.7.3
```
At present, the target of this PR is master branch. Is this right?
If I had to choose a target, I like master branch a bit because this is not a obvious bug.

jfbu replied:
@tk0miya Yes this is right. I chose master branch because it is not obvious bug.

jfbu replied:
It is only default, and user can override it already to fix it without waiting for 1.8. (long time ago it was not customizable)

jfbu replied:
ah sorry I see what you mean, ok I will edit to 1.8, OMG!

jfbu replied:
fixed...

tk0miya replied:
Confirmed. Go ahead :-)

jfbu replied:
Thanks @tk0miya, now merging into master.

jfbu committed:
Typo (Sphinx version number wrong in documentation of ``'fvset'`` key)

jfbu committed:
Fix LaTeX style file version number

(I should perhaps have left it standing, as this may cause merge
conflicts if merged after long delay)
